<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>29</y>
     <width>591</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="ImageLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="GlobalTBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Global Thresholding (V = 127)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdaptiveMBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptive Mean Thresholding</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdaptiveGBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptive Gaussian Thresholding</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GaussianBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gaussian Blur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MedianBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Median Blur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OriginalBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputPath">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Test Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modelOneBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test With Model - 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modelTwoBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test With Model - 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>771</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
